test/services/AppealServiceSpec.scala

```diff
@@ -50,7 +50,21 @@ class AppealServiceSpec extends SpecBase {
     SessionKeys.hasConfirmedDeclaration -> true,
     SessionKeys.dateOfCrime -> LocalDate.parse("2022-01-01"),
     SessionKeys.penaltyNumber -> "123456789",
-    SessionKeys.appealType -> PenaltyTypeEnum.Late_Submission
+    SessionKeys.appealType -> PenaltyTypeEnum.Late_Submission,
+    SessionKeys.doYouWantToAppealBothPenalties -> "no"
+  )))(fakeRequest)
+
+  val fakeRequestForCrimeJourneyMultiple: UserRequest[AnyContent] = UserRequest("123456789", answers = userAnswers(Json.obj(
+    SessionKeys.reasonableExcuse -> "crime",
+    SessionKeys.dateCommunicationSent -> LocalDate.parse("2021-12-01"),
+    SessionKeys.hasCrimeBeenReportedToPolice -> "yes",
+    SessionKeys.hasConfirmedDeclaration -> true,
+    SessionKeys.dateOfCrime -> LocalDate.parse("2022-01-01"),
+    SessionKeys.penaltyNumber -> "123456789",
+    SessionKeys.appealType -> PenaltyTypeEnum.Late_Submission,
```

Comment:
Wouldn't this be the case for LPP and Additional

Reply:
Updated in accordance with new logic